apps/etterlatte-pdltjenester/src/main/kotlin/person/PersonService.kt

```diff
@@ -91,13 +92,24 @@ class PersonService(
                 }
             } else {
                 try {
-                    val folkeregisterIdent: String = it.data.hentIdenter.identer
+                    val folkeregisterIdent: String? = identResponse.data.hentIdenter.identer
                         .filter { it.gruppe == PDLIdentGruppeTyper.FOLKEREGISTERIDENT.navn }
-                        .first { !it.historisk }.ident
-                    FolkeregisterIdent(folkeregisterident = Folkeregisteridentifikator.of(folkeregisterIdent))
+                        .firstOrNull { !it.historisk }?.ident
+                    if (folkeregisterIdent != null) {
+                        PdlIdentifikator.FolkeregisterIdent(
+                            folkeregisterident = Folkeregisteridentifikator.of(
+                                folkeregisterIdent
+                            )
+                        )
+                    } else {
+                        val npid: String = identResponse.data.hentIdenter.identer
```

Comment:
nitpick `NPID`?

Reply:
eh, foretrekker navnekonvensjoner om at variabler er lowercase egentlig, og at caps er for `const val`

Reply:
aight

Reply:
så nPId eller nPID, da. Vi kjører vel lowerCamelCase :P